starter-code/views/signin.ejs

```diff
@@ -0,0 +1,16 @@
+
+<form method="post" action="/signin">
+  <div>
+    <label for="username"> Your username: </label>
+    <input type="text" name="username">
+  </div>
+
+  <div>
+    <label for="password"> Your password: </label>
+    <input type="password" name="password">
+  </div>
+  <% if (typeof(errorMessage) != "undefined") { %>
```

Comment:
note1: `typeof` requires no parenthesis around the var

note2: you could also write a simple "truthy" check `if (errorMessage) { ... `

note3: if you are going to compare anything for inequality, you should use `!==` instead of `!=`